index.js

```diff
@@ -31,4 +31,158 @@ navLinks.forEach((link) => {
+    tech: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    imageMob: 'Images/Snapshoot3.png',
+    imageDesk: 'Images/Desktop3.png',
+    live: '#',
+    source: '#',
+  },
+  {
+    id: 4,
+    title: 'Multi-Post Stories',
+    canopy: ['CANOPY', 'Back End Dev', '2015'],
+    dot: './img/Counter.png',
+    class: 'desk-content',
+    para1: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    para: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and. It has survived not only five centuries.",
+    para2: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    tech: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    imageMob: 'Images/Snapshoot4.png',
+    imageDesk: 'Images/Desktop-4.png',
+    live: '#',
+    source: '#',
+  },
+];
+
+const sectionLabel = document.getElementById('work');
+
+workCards.forEach((item2) => {
+  sectionLabel.innerHTML += `
+  <div class="${item2.class}">
+    <img class="portfolios" src="${item2.imageMob}" alt="card1">
```

Comment:
- [ OPTIONAL ] I highly recommend to decrease the width of the image since I think it is out of the pop up a little bit so please just decrease the size ot the image 💯🔥♻💪🏻because in small screens it get out of the frame 😊🙏🏻